crates/proof-of-sql/src/base/database/group_by_util_test.rs

```diff
@@ -335,11 +498,12 @@ fn we_can_sum_aggregate_columns_by_counts() {
     let columns_b = Column::Int128::<Curve25519Scalar>(slice_b);
     let columns_c = Column::Scalar(&scals_c);
     let indexes = &[12, 11, 1, 10, 2, 3, 6, 14, 13, 9];
-    let counts = &[3, 3, 4];
+    let counts = &[3, 3, 4, 0];
     let expected = &[
         Curve25519Scalar::from(112 + 111 + 101),
         Curve25519Scalar::from(110 + 102 + 103),
         Curve25519Scalar::from(106 + 114 + 113 + 109),
+        Curve25519Scalar::from(0),
```

Comment:
Can we keep this test as it was since this behavior is unneeded and tested above?